### DISCUSSION TITLE
blur the image

### DISCUSSION THREAD

**Discussion by Gutha-Vamsi-Krishna** [skydoves/landscapist#82] [Q&A] [answered]
Is there way to blur the image loaded using this library?

**Comment by skydoves** [comment]
Hey @Gutha-Vamsi-Krishna,
Do you mean blurring while loading images or blurring a result of the image?

**Reply by Gutha-Vamsi-Krishna** [reply]
What I was doing is that, first i am loading high quality image, while it takes time to load, instead of showing spinner, i was loading same image with low quality, So I want to apply blur to this low quality image.

**Reply by skydoves** [reply]
https://github.com/skydoves/landscapist#blurtransformationplugin

**Comment by skydoves** [comment, marked as answer]
https://github.com/skydoves/landscapist#blurtransformationplugin
